Source/WebKit/UIProcess/API/Cocoa/WKUIDelegatePrivate.h

```diff
@@ -255,6 +256,9 @@ struct UIEdgeInsets;
 - (NSArray<UIDragItem *> *)_webView:(WKWebView *)webView willPerformDropWithSession:(id <UIDropSession>)session WK_API_AVAILABLE(ios(11.0));
 - (NSInteger)_webView:(WKWebView *)webView dataOwnerForDropSession:(id <UIDropSession>)session WK_API_AVAILABLE(ios(11.0));
 - (NSInteger)_webView:(WKWebView *)webView dataOwnerForDragSession:(id <UIDragSession>)session WK_API_AVAILABLE(ios(11.0));
+// FIXME (rdar://96239605): Expose these edit menu delegates as public API.
```

Comment:
Technically I think this should point at a non-opaque bugzilla bug (though I am guilty of sometimes pasting radars too)

Reply:
Will do! Filed: https://bugs.webkit.org/show_bug.cgi?id=243102 (and I'll replace the rdar:// link with a bugzilla link).